mysql-test/suite/component_keyring_kmip/t/tablespace_encrypt_6.test

```diff
@@ -0,0 +1,6 @@
+--source include/have_component_keyring_kmip.inc
```

Comment:
I accidentally added these back somehow, after removing them first - these tests do not work with KMIP, because they either assume that they can take a backup by copying the keyring file, or that they can use two different keyrings. Removed.